### PR TITLE
Bump schedulers.config.openshift.io CRD to v1

### PR DESCRIPTION
**Pull request by ingvagabund**
v1beta1 does not know `default` keyword

### CODE REVIEW

**Review by damemi — COMMENTED** [review]
/lgtm

**Comment by ingvagabund** [comment]
Once merged, the changes need to go into https://github.com/openshift/cluster-config-operator/pull/168

File: config/v1/0000_10_config-operator_01_scheduler.crd.yaml [hunk 1]
```diff
@@ -19,89 +19,90 @@ spec:
   - name: v1
     served: true
     storage: true
+    "schema":
```
Comment (sttts) [inline config/v1/0000_10_config-operator_01_scheduler.crd.yaml:22]: remove preserveUnknownFields above

File: config/v1/0000_10_config-operator_01_scheduler.crd.yaml [hunk 2]
```diff
@@ -19,89 +19,90 @@ spec:
...
+                enum:
+                - ""
+                - LowNodeUtilization
+                - HighNodeUtilization
+                - NoScoring
+          status:
+            description: status holds observed values from the cluster. They may not
+              be overridden.
+            type: object
   subresources:
```
Comment (sttts) [inline config/v1/0000_10_config-operator_01_scheduler.crd.yaml:107]: move this under the version

File: config/v1/0000_10_config-operator_01_scheduler.crd.yaml (hunk 1 above)
Comment (damemi) [reply config/v1/0000_10_config-operator_01_scheduler.crd.yaml:22]: /lgtm cancel
btw, we check this isn't set in verify-crds.sh but only for v1beta1 CRDs. should we expand that to check that it's not set in v1? I thought that wasn't even available in v1

Comment (sttts) [reply config/v1/0000_10_config-operator_01_scheduler.crd.yaml:23]: Yes, please.

Also top-level subresources is a common error. I also did it in the past.

**Pull request merged**